### PR TITLE
added python classes data

### LINKED ISSUE #273: Add Python Classes
# Description

Classes for Python aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/classes.json` to the right Python directory in `web/thesauruses/`
* Edit the classes.json file to accurately reflect the latest major version of Python
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by amitpant14**
## What GitHub issue does this PR apply to?

Resolves #273 

## What changed and why?

Added data for python classes.

## Checklist

- [X] I claimed any associated issue(s) and they are not someone else's
- [X] I have looked at the documentation to ensure I made any revisions correctly
- [X] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/python/classes.json [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    "normal_class": {
+      "code": "class class_name:\n  # class body"
+    },
+    "abstract_class": {
+      "code": "from abc import ABC\nclass class_name(ABC):\n  # class body",
+      "comment": "Note that you will need to import ABC in order to be able to use abstract classes."
+    },
+    "interface": {
+      "code": "import abc\nclass class_name:\n   # Way 1\n   def method1(self):\n      # pass\n   # Way 2\n   @abc.abstractmethod\n   def method2(self):\n      raise NotImplementedError",
+      "comment": "Note that way 1 doesn't force the user to define the methods from the abstract class in the derived class, while way 2 enforces that."
```
Comment (geekygirlsarah) [inline web/thesauruses/python/classes.json:53]: My first thought was "wait, you're just using 'abc' as an example. You should use something real."  Nope. It's Abstract Base Classes, lol. I never knew this!

File: web/thesauruses/python/classes.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    },
+    "read_only_class": {
+      "code": "",
+      "not-implemented": true
+    },
```
Comment (geekygirlsarah) [inline web/thesauruses/python/classes.json]: If something's not implemented, you can remove the code line. (it cleans it up a bit.) You have a few of those in this file.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
Just do the tiny change to clean up the file and I'll merge it in!

File: web/thesauruses/python/classes.json (hunk 2 above)
Comment (amitpant14) [reply web/thesauruses/python/classes.json]: Sure.

**Review by geekygirlsarah — APPROVED** [review]

**Comment by geekygirlsarah** [comment]
Great! Thanks for the updates, and this looks good to go now. I'll merge it in!

**Pull request merged**